docs/streams.html

```diff
@@ -790,13 +790,15 @@
         </p>
         <br>
 
-        <h2><a id="streams_upgrade" href="#streams_upgrade">Upgrade Guide and API Changes</a></h2>
+        <h2><a id="streams_upgrade_and_api_0102" href="#streams_upgrade_and_api_0102">Upgrade Guide and API Changes</a></h2>
```

Comment:
Do we want to make this section specifically for 0.10.2 or just make it a general section which contains links to the upgrade sections for each version?